Bindings/Java/Matlab/Dynamic_Walker_Example/Main_WalkerForwardSimWithControls.m

```diff
@@ -0,0 +1,78 @@
+% ----------------------------------------------------------------------- 
+% The OpenSim API is a toolkit for musculoskeletal modeling and           
+% simulation. See http://opensim.stanford.edu and the NOTICE file         
+% for more information. OpenSim is developed at Stanford University       
+% and supported by the US National Institutes of Health (U54 GM072970,    
+% R24 HD065690) and by DARPA through the Warrior Web program.             
+%                                                                         
+% Copyright (c) 2005-2013 Stanford University and the Authors             
+% Author(s): Daniel A. Jacobs                                             
+%                                                                         
+% Licensed under the Apache License, Version 2.0 (the "License");         
+% you may not use this file except in compliance with the License.        
+% You may obtain a copy of the License at                                 
+% http://www.apache.org/licenses/LICENSE-2.0.                             
+%                                                                         
+% Unless required by applicable law or agreed to in writing, software     
+% distributed under the License is distributed on an "AS IS" BASIS,       
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         
+% implied. See the License for the specific language governing            
+% permissions and limitations under the License.                          
+% -----------------------------------------------------------------------
+% DesignMainStarterWithControls
+%   This script opens a Model, edits its initial state, runs a forward 
+% simulation with the built-in Matlab integrators, and creates a plot from 
+% the motion data.  The model has an coordinate actuator on the left knee
+% and a position controller to set the angle to -135 degrees.
+
+% Make sure to run the AddCoordinateActuator.m script first!
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+display('Make sure you run the script file "AddCoordinateActuator.m" first.')
```

Comment:
Instead, perhaps read the model and throw an error if the expected components are missing?